plugins/oracledb_metrics.yaml

```
@@ -0,0 +1,649 @@
+version: 0.0.1
+title: OracleDB Metrics
+description: Metrics receiver for OracleDB
+parameters:
+  - name: endpoint
+    description: Address to scrape metrics from
+    type: string
+    default: localhost:1521
+  - name: username
+    description: Database user to run metric queries with
+    type: string
+    required: true
+  - name: password
+    description: Password for user
+    type: string
+  - name: sid
+    description: Site Identifier. One of sid or service_name must be specified both can be.
+    type: string
+  - name: service_name
+    description: OracleDB Service Name. One of sid or service_name must be specified both can be.
+    type: string
+  - name: wallet
+    description: OracleDB Wallet file location (must be URL encoded)
+    type: string
+  - name: scrape_interval
+    description: Time in between every scrape request
+    type: string
+    default: 60s
+template: |
+  # construct datasource endpoint based on config
+  {{$datasource := "oracle://"}}
+  {{ $datasource = printf "%s%s" $datasource .username }}
+  {{ if .password }}
+    {{ $datasource = printf "%s:%s" $datasource .password }}
+  {{end}}
+  {{ $datasource = printf "%s@%s" $datasource .endpoint }}
+  {{ if .service_name }}
+    {{ $datasource = printf "%s/%s" $datasource .service_name }}
+  {{end}}
+        {{ if and .sid .wallet }}
+    {{ $datasource = printf "%s?SID%%3D%s&WALLET%3D%s" $datasource .sid .wallet}}
```

I think this is missing an escape?
```suggestion
    {{ $datasource = printf "%s?SID%%3D%s&WALLET%%3D%s" $datasource .sid .wallet}}
```